src/EditorFeatures/Core/BackgroundWorkIndicator/BackgroundWorkIndicatorScope.cs

```diff
@@ -3,87 +3,102 @@
 // See the LICENSE file in the project root for more information.
```

Comment:
this file is effectively a rewrite.  I would look at the SxS view, not hte inline view.  And really just pay attention to how the new code is written.